web/src/styles/scss/secondary-nav.scss

```diff
@@ -1,11 +1,11 @@
 @import '@cmsgov/design-system-support/src/settings/index';
 
-.next-prev-buttons {
+.pre-button-section-break {
   margin: 32px 0px;
   padding-top: 32px;
```

Comment:
Much of the rest of the app uses `$spacer-x` when declaring margin and padding values. Any chance we could use those, here?

`$spacer-1` is defined as `8px`, so you want `$spacer-4` in place of `32px`

Reply:
thanks for the suggestion, I hadn't noticed the $spacer-x variables